pyproject.toml

```diff
@@ -54,12 +54,14 @@ uvicorn = {version = "^0.24.0.post1", optional = true}
 chromadb = "^0.4.18"
 pytest-asyncio = {version = "^0.23.2", optional = true}
 pydantic = "^2.5.2"
+pyautogen = {version = "0.2.0", optional = true}
```

Comment:
Just confirming - did you generate this + the lock file by doing:

```
poetry add pyautogen==0.2.0
```

Reply:
yes, this entry was addded by poetry